src/libraries/Common/src/System/Security/Cryptography/Asn1/asn.xslt

```diff
@@ -47,6 +47,13 @@
     </xsl:choose>
   </xsl:variable>
 
+  <xsl:variable name="hasRebind" xml:space="default">
```

Comment:
I'd much rather that we just change the XSLT to suppress the code, and undo the changes to all the ASN xml files.

Reply:
just curious why? I'd need to update all the files and there seems to be a lot of them

Reply:
Well, mainly it's an automatic response to the combination of "we're just doing this thing in bulk" and "it makes the generator more complicated".

If something which has had rebind turned off gains a rebinding field that'll fail the build, which just means you have to turn rebind-off off, and that might cascade... but... I guess that's all OK (and seems unlikely).  So... I guess it's OK.

Reply:
GitHub won't allow me to attach this comment in the right place, but this is close enough:

I'm 98% sure that all of these `$hasRebind &gt; 0` can just be reduced to `$hasRebind`, since your only values are 0 and 1.

xsl:if's test expression evaluator is the same as the XPath expression evaluator, which says (https://www.w3.org/TR/xpath20/#id-ebv)

> If its operand is a singleton value of any numeric type ... returns false if the operand value is NaN or is numerically equal to zero; otherwise it returns true.

Reply:
I agree that this should be possible and I tried but could not get it working (for some reason the boolean conversion always evaluates to true).